File: 프로그래머스/lv2/12939. 최댓값과 최솟값/최댓값과 최솟값.cpp
#include <iostream>
#include <string>
#include <vector>
#include <algorithm>
#include <sstream>

using namespace std;

string solution(string s) {
	string answer = "";
	vector<int>num;
	stringstream ss(s);
	string str;
	while (ss >> str) {
		num.push_back(stoi(str));
	}
	int max = *max_element(num.begin(), num.end());
	int min = *min_element(num.begin(), num.end());
	answer += to_string(min);
	answer += " ";
	answer += to_string(max);

	return answer;
}

File: 프로그래머스/2/12939. 최댓값과 최솟값/최댓값과 최솟값.py
def solution(s):
    arr = s.split(' ')
    min_number = int(arr[0])
    max_number = int(arr[0])
    for i in range(0,len(arr)):
        min_number = min(int(arr[i]),min_number)
        max_number = max(int(arr[i]),max_number)
    answer = f'{min_number} {max_number}'
    return answer